crates/proof-of-sql/src/sql/proof/provable_query_result.rs

```diff
@@ -14,7 +12,6 @@ use serde::{Deserialize, Serialize};
 #[derive(Default, Clone, Serialize, Deserialize)]
 pub struct ProvableQueryResult {
     num_columns: u64,
-    indexes: Indexes,
```

Comment:
Could we replace this with `table_length: usize` instead? That would allow `to_owned_table` to stay unchanged.